Updated OpenGraph title to display folder name

mastix opened the pull request:
When sharing a folder via Facebook, WhatsApp & Co (any platform that makes use of OpenGraph) the display title was "Nextcloud - a safe home for all your data" and the description was the actual name of the folder. This should not be the case, so I changed the display title to display the folder name and the description now displays the form OpenGraph title. Google and Dropbox actually work the same way.

It looks much better now. :)

mastix committed:
Updated OpenGraph title to display folder name

When sharing a folder via Facebook, WhatsApp & Co (any platform that makes use of OpenGraph) the display title was "Nextcloud - a safe home for all your data" and the description was the actual name of the folder. This should not be the case, so I changed the display title to display the folder name and the description now displays the form OpenGraph title.

MorrisJobke commented on apps/files_sharing/lib/Controller/ShareController.php:
```diff
@@ -406,12 +406,13 @@ public function showShare($token, $path = '') {
+        	\OCP\Util::addHeader('meta', ['property' => "og:site_name", 'content' => $this->defaults->getName()]);
+        	\OCP\Util::addHeader('meta', ['property' => "og:url", 'content' => $shareTmpl['shareUrl']]);
+        	\OCP\Util::addHeader('meta', ['property' => "og:type", 'content' => "object"]);
+        	\OCP\Util::addHeader('meta', ['property' => "og:image", 'content' => $shareTmpl['previewImage']]);
```
Looks like you replaced the tabs here with spaces. Could you revert that to tabs please? Then the diff should also look better.

MorrisJobke commented on apps/files_sharing/lib/Controller/ShareController.php:
```diff
@@ -406,12 +406,13 @@ public function showShare($token, $path = '') {
+        	\OCP\Util::addHeader('meta', ['property' => "og:url", 'content' => $shareTmpl['shareUrl']]);
+        	\OCP\Util::addHeader('meta', ['property' => "og:type", 'content' => "object"]);
+        	\OCP\Util::addHeader('meta', ['property' => "og:image", 'content' => $shareTmpl['previewImage']]);
+
```
Please also remove this added line. There is already an empty line here.

MorrisJobke reviewed (changes requested):
Looks good beside the whitespace changes.

mastix commented:
Ok - lesson learned: Never fix a bug on Github directly. I'll use my IDE and will try it again.

mastix committed:
Fixing whitespaces and empty line.

mastix commented:
Updated PR.

MorrisJobke reviewed (approved):
Tested and works 👍

MorrisJobke commented:
One more thing: all commit need a sign-off message. Check out https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work to know why and how. Best would be to squash the two commits and add this sign-off message.

Thanks

mastix committed:
Updated OpenGraph title to display folder name instead of a nextcloud standard text

When sharing a folder via Facebook, WhatsApp or any other platform that makes use of OpenGraph, the display title was "Nextcloud - a safe home for all your data" and the description was the actual name of the folder. This should not be the case, so the display title has been changed to display the folder name and the description now displays the former OpenGraph title.

Signed-off-by: Sascha Sambale <mastixmc@gmail.com>

mastix committed:
Merge remote-tracking branch 'origin/master'

ChristophWurst reviewed (approved):
Makes sense and changes look good

MorrisJobke commented:
Let me take care of squashing and all 😉

MorrisJobke commented:
Merge as squashed version 30460b52df via 4cb4d03184

MorrisJobke commented:
Thanks for your contribution @mastix 👍 🚀 

Maybe you want to look into other small issues - we collect them under the starter issue label: https://github.com/nextcloud/server/issues?q=is%3Aissue+is%3Aopen+label%3A%22starter+issue%22
